lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1299,12 +1300,18 @@ def setUp(self):  # pylint: disable=arguments-differ
         ('student_enrolled', 403),
         ('staff', 204),
         ('course_staff', 204),
-        ('community_ta', 204)
+        ('community_ta', 204),
+        ('admin', 204)
     )
     @ddt.unpack
     def test_access(self, user, status):
+        if user is not None:
+            team_list = self.get_teams_list(user='course_staff', expected_status=200)
```

Comment:
@Ali-D-Akbar why are you hardcoding the `user` attribute here?

Reply:
It takes a user which is the owner of that team. This is why I have hardcoded it. There are many instances where a hardcoded user has been used.